pypdf/generic/_data_structures.py

```diff
@@ -121,6 +128,43 @@ def items(self) -> Iterable[Any]:
         """Emulate DictionaryObject.items for a list (index, object)."""
         return enumerate(self)
 
+    def __to_lst(self, lst: Any) -> List[Any]:
```

Comment:
* Why is `_to_list` not sufficient? A leading underscore already indicates a private implementation detail and using the verbose name should not hurt in this case.
* And why do we need to make this a regular method instead of a function or class method? I see no actual reference to the object itself.